vips/vip-Gateway/bscmainnet.ts

```diff
@@ -0,0 +1,299 @@
+    holder: "0xf89d7b9c864f589bbf53a82105107622b35eaa40",
+  },
+  {
+    name: "vSXP",
+    address: "0x2ff3d0f6990a40261c66e1ff2017acbc282eb6d0",
+    holder: "0x44cf30ea4e58818bfae4b8499be409cd4fdd5a20",
+  },
+  {
+    name: "vTRX",
+    address: "0xc5d3466aa484b040ee977073fcf337f2c00071c1",
+    holder: "0xf977814e90da44bfa03b6295a0616a897441acec",
+  },
+  {
+    name: "vTRXOLD",
+    address: "0x61edcfe8dd6ba3c891cb9bec2dc7657b3b422e93",
+    holder: "0xe2fc31f816a9b94326492132018c3aecc4a93ae1",
+  },
+  {
+    name: "vTUSD",
+    address: "0xbf762cd5991ca1dcddac9ae5c638f5b5dc3bee6e",
+    holder: "0xf977814e90da44bfa03b6295a0616a897441acec",
+  },
+  {
+    name: "vTUSDOLD",
+    address: "0x08ceb3f4a7ed3500ca0982bcd0fc7816688084c3",
+    holder: "0x2e28b9b74d6d99d4697e913b82b41ef1cac51c6c",
+  },
+  {
+    name: "vUNI",
+    address: "0x27ff564707786720c71a2e5c1490a63266683612",
+    holder: "0x0000000000000000000000000000000000001004",
+  },
+  {
+    name: "vUSDC",
+    address: "0xeca88125a5adbe82614ffc12d0db554e2e2867c8",
+    holder: "0x8894E0a0c962CB723c1976a4421c95949bE2D4E3",
+  },
+  {
+    name: "vUSDT",
+    address: "0xfd5840cd36d94d7229439859c0112a4185bc0255",
+    holder: "0xee5b5b923ffce93a870b3104b7ca09c3db80047a",
+  },
+  {
+    name: "vWBETH",
+    address: "0x6cfdec747f37daf3b87a35a1d9c8ad3063a1a8a0",
+    holder: "0xf977814e90da44bfa03b6295a0616a897441acec",
+  },
+  {
+    name: "vXRP",
+    address: "0xb248a295732e0225acd3337607cc01068e3b9c10",
+    holder: "0x5a52e96bacdabb82fd05763e25335261b270efcb",
+  },
+  {
+    name: "vXVS",
+    address: "0x151b1e2635a717bcdc836ecd6fbb62b674fe3e1d",
+    holder: "0xfd36e2c2a6789db23113685031d7f16329158384",
+  },
+];
+
+const REWARD_DISTRIBUTORS_STABLECOIN = [
+  "0xBA711976CdF8CF3288bF721f758fB764503Eb1f6",
+  "0xA31185D804BF9209347698128984a43A67Ce6d11",
+  "0x177ED4625F57cEa2804EA3A396c8Ff78f314F1CA",
+];
+
+const REWARD_DISTRIBUTORS_DEFI = [
+  "0x7524116CEC937ef17B5998436F16d1306c4F7EF8",
+  "0x14d9A428D0f35f81A30ca8D8b2F3974D3CccB98B",
+  "0xD86FCff6CCF5C4E277E49e1dC01Ed4bcAb8260ba",
+];
+
+const REWARD_DISTRIBUTORS_GAMEFI = [
+  "0x501a91b995Bd41177503A1A4144F3D25BFF869e1",
+  "0x2517A3bEe42EA8f628926849B04870260164b555",
+];
+
+const REWARD_DISTRIBUTORS_LST = [
+  "0x63aFCe42086c8302659CA0E21F4Eade27Ad85ded",
+  "0x79397BAc982718347406Ebb7A6a8845896fdD8dE",
+  "0x6a7b50EccC721f0Fa9FD7879A7dF082cdA60Db78",
+  "0xBE607b239a8776B47159e2b0E9E65a7F1DAA6478",
+  "0x888E317606b4c590BBAD88653863e8B345702633",
+];
+
+const REWARD_DISTRIBUTORS_TRON = [
+  "0x804F3893d3c1C3EFFDf778eDDa7C199129235882",
+  "0x6536123503DF76BDfF8207e4Fb0C594Bc5eFD00A",
+  "0x22af8a65639a351a9D5d77d5a25ea5e1Cf5e9E6b",
+  "0x08e4AFd80A5849FDBa4bBeea86ed470D697e4C54",
+];
+
+const ALL_REWARD_DISTRIBUTORS = [
+  ...REWARD_DISTRIBUTORS_DEFI,
+  ...REWARD_DISTRIBUTORS_GAMEFI,
+  ...REWARD_DISTRIBUTORS_LST,
+  ...REWARD_DISTRIBUTORS_STABLECOIN,
+  ...REWARD_DISTRIBUTORS_TRON,
+];
```

Comment:
These RewardDistributor contracts don't exist on Ethereum. I would remove it from here, and I would use the wildcard ZERO_ADDRESS to allow the Guardian to execute the new function on every contract

Reply:
Done https://github.com/VenusProtocol/vips/pull/195/commits/45a2e5bde58b675a6ab6cf28ba03b2efd045a7e4